### DISCUSSION TITLE
apollo迁移

### DISCUSSION THREAD

**Discussion by FaxonXing** [apolloconfig/apollo#3957] [Q&A] [answered]
我们尝试将apollo迁移到k8s上，并保留旧数据。我们按照文档中的说明对下面四个表进行了迁移
![apollo文档](https://user-images.githubusercontent.com/88068157/132298060-8850624e-c7e4-4aca-aa5a-81ccf04e9a6c.png)

这样在访问面板时，很多数据都不存在；然后我们将所有数据（在应用启动后同步，否则启动不了）都同步过来，这样面板上出现了部分数据，但还是有缺失。我该如何解决？ApolloConfigDB和ApolloPortalDB中存在一些相同的表，他们是如何工作的呢？

希望得到大佬的帮助，万分感谢

**Comment by nobodyiam** [comment, marked as answer]
文档中的说明是针对增加新环境的，如果只是平迁的话（没有增加新环境），那么直接把数据库全量迁移即可，相当于是把数据平迁
